test/index.spec.js

```diff
@@ -361,7 +361,7 @@ describe('CID', () => {
 
       // it cached!
       expect(cid.string).to.equal(base32String)
-      expect(Object.hasOwnProperty('_string')).to.be.false()
+      expect(Object.prototype.hasOwnProperty.call(Object, '_string')).to.be.false()
```

Comment:
Same here, I don't understand the purpose of this assertion. `_string` is not used in the code ever.